infrastructure/workers-configuration/workers-instance-user-data.tpl.sh

```diff
@@ -0,0 +1,71 @@
+MIME-Version: 1.0
+Content-Type: multipart/mixed; boundary="==MYBOUNDARY=="
```

Comment:
Good question. I forget exactly why I ended up doing this, but I think it was when I was struggling to get the instances to run the jobs and this maybe helped? It might not have helped at all though... I'm not entirely sure why but I think that maybe Batch wouldn't run the script unless it was like this.

I know this is working though!